controllers/profiles/reconciler_dev.go

```diff
@@ -120,9 +127,30 @@ func (e *ensureRunningDevWorkflowReconciliationState) Do(ctx context.Context, wo
 	}
 	objs = append(objs, propsCM)
 
+	devBaseContainerImage := defaultKogitoServerlessWorkflowDevImage + ":" + version.OperatorVersion
+	pl, errPl := platform.GetActivePlatform(ctx, e.client, workflow.Namespace)
+	// check if the Platform available
+	if errPl == nil {
+
+		// if is present the optional devBaseImage
+		if len(pl.Spec.DevBaseImage) > 0 {
+			devBaseContainerImage = pl.Spec.DevBaseImage
+		} else {
+			// If is not present the optional devBaseImage and is present the baseImage
+			if len(pl.Spec.BuildPlatform.BaseImage) > 0 {
+				devBaseContainerImage = pl.Spec.BuildPlatform.BaseImage
```

Comment:
If there isn't an override in the Platform, why are we using the Base Build Image?

Reply:
changed